### DISCUSSION TITLE
Welcome to RestApi Discussions!

### DISCUSSION THREAD

**Discussion by DevopsTech10** [DevopsTech10/RestApi#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DevopsTech10** [comment]
##how i can improve this api any tips

**Reply by MuradCade** [reply]
improve database query

**Comment by MuradCade** [comment]
reduce the request in database
